assets/scss/globals/_google-font.scss

```diff
@@ -0,0 +1,126 @@
+/* latin-ext */
+@font-face {
+  font-family: 'Open Sans';
+  font-style: italic;
+  font-weight: 400;
+  font-display: swap;
+  src: url(https://fonts.gstatic.com/s/opensans/v20/mem6YaGs126MiZpBA-UFUK0Xdc1GAK6bt6o.woff2) format('woff2');
```

Comment:
Le CSS est local mais il y a toujours un appel au fichier de la police elle-même, du coup ça ne fait que déplacer le problème non ? Est-ce qu'il ne serait pas possible de mettre le fichier woff en local aussi ? 
https://gomakethings.com/how-to-self-host-google-fonts/

Reply:
Hello
Oui, je m'en suis aperçu qu'après. 🤦‍♂️ D'où le repassage en WIP, je m'en occupe ASAP. 🙂